content/operate/rc/changelog/may-2024.md

```diff
@@ -7,17 +7,23 @@ categories:
 - rc
 description: New features, enhancements, and other changes added to Redis Cloud during
   May 2024.
-highlights: Active-Active search support
+highlights: SAML domain verification, Active-Active search support
 linktitle: May 2024
 weight: 50
 ---
 
 ## New features
 
+### SAML domain verification
+
+We added domain verification to [SAML single sign-on]({{< relref "/operate/rc/security/access-control/saml-sso" >}}) to enhance security. Before you set up SAML in Redis Cloud, you must first [verify domain ownership]({{< relref "/operate/rc/security/access-control/saml-sso#verify-domain" >}}) for any domains associated with your SAML setup.
+
+You can now also associate multiple domains with your SAML setup.
```

Comment:
@cmilesb I think this warrants a section of it own since this is a big benefit for larger customers that are using SAML. Something like: "SAML SSO now supports multiple username  domains"